css/classPage.css

```diff
@@ -0,0 +1,56 @@
+[h3^="#class-title-"] {
+    background-color: blue;
+}
+
+.tableWrapper {
+    overflow: hidden;
+    transition: max-height 0.5s ease-in-out;
+}
+
+div[data-expand="true"] {
+    max-height: 5000px; /* magic number for now hopefully :/ (I NEED CSS4 NOW) */
```

Comment:
I'm using a magic number here so that we are able to get collapse/expand behavior. Should we drop this effect and just have class tables pop out?